test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -479,7 +480,7 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI [Privileged:ClusterAdmin]", fu
 	})
 
 	// Marked as flaky until https://github.com/kubernetes/kubernetes/issues/65517 is solved.
```

Comment:
the issue referenced here is fixed (https://github.com/kubernetes/kubernetes/issues/65517) but the test is solidly failing in https://testgrid.k8s.io/google-gce#gce-cos-master-flaky-repro

seeing if there's something else going on

Reply:
changes made to explain since that fix:
* https://github.com/kubernetes/kubernetes/pull/91295